crates/orderbook/src/database/orders.rs

```diff
@@ -305,6 +305,17 @@ impl OrderStoring for Postgres {
                     if let Some(quote) = new_quote {
                         insert_quote(&new_order.metadata.uid, &quote, ex).await?;
                     }
+                    if let Some(full_app_data) = new_order.metadata.full_app_data {
```

Comment:
This is a 1:1 copy of the same logic inside `insert_order()`, right?
In that case moving that into a separate function would make sense to me.